README.md

```diff
@@ -32,11 +32,11 @@ To build a Liferay Faces Bridge demo from source:
 
 2. Build the demo for Liferay:*
 
-		mvn clean install -P liferay
+		mvn clean package -P liferay,liferay-portlet-war-redeploy,tomcat,war-license-packaging,activate-portlet-war,faces-war,mojarra,redeploy
```

Comment:
@stiemannkj1 When we release the demo portlets, we use the following command:
```mvn --batch-mode -P release,liferay,mojarra,tomcat```
Since readers of the README aren't actually "releasing", I think the following would be adequate:
```mvn --batch-mode -P liferay,mojarra,tomcat```